data/yam/autoyast/create_hdd_textmode_xen_hvm.xml

```diff
@@ -0,0 +1,325 @@
+      <secure_boot>true</secure_boot>
+      <terminal>gfxterm</terminal>
+      <timeout t="integer">30</timeout>
+      <trusted_grub>false</trusted_grub>
+      <update_nvram>true</update_nvram>
+      <xen_kernel_append>vga=gfx-1024x768x16</xen_kernel_append>
+    </global>
+    <loader_type>grub2</loader_type>
+  </bootloader>
+  <firewall t="map">
+    <default_zone>public</default_zone>
+    <enable_firewall t="boolean">true</enable_firewall>
+    <log_denied_packets>off</log_denied_packets>
+    <start_firewall t="boolean">true</start_firewall>
+    <zones t="list">
+      <zone t="map">
+        <description>For use in public areas. You do not trust the other computers on networks to not harm your computer. Only selected incoming connections are accepted.</description>
+        <interfaces t="list">
+          <interface>eth0</interface>
+        </interfaces>
+        <masquerade t="boolean">false</masquerade>
+        <name>public</name>
+        <ports t="list"/>
+        <protocols t="list"/>
+        <services t="list">
+          <service>dhcpv6-client</service>
+        </services>
+        <short>Public</short>
+        <target>default</target>
+      </zone>
+    </zones>
+  </firewall>
+  <general t="map">
+    <mode t="map">
+      <confirm t="boolean">false</confirm>
+    </mode>
+  </general>
+  <groups t="list">
+    <group t="map">
+      <gid>100</gid>
+      <groupname>users</groupname>
+      <userlist/>
+    </group>
+  </groups>
+  <networking t="map">
+    <dhcp_options t="map">
+      <dhclient_client_id/>
+      <dhclient_hostname_option>AUTO</dhclient_hostname_option>
+    </dhcp_options>
+    <dns t="map">
+      <dhcp_hostname t="boolean">true</dhcp_hostname>
+      <hostname>prg2e-d74</hostname>
+      <resolv_conf_policy>auto</resolv_conf_policy>
+    </dns>
+    <interfaces t="list">
+      <interface t="map">
+        <bootproto>dhcp</bootproto>
+        <name>eth0</name>
+        <startmode>auto</startmode>
+        <zone>public</zone>
+      </interface>
+    </interfaces>
+    <ipv6 t="boolean">true</ipv6>
+    <keep_install_network t="boolean">true</keep_install_network>
+    <managed t="boolean">false</managed>
+    <net-udev t="list">
+      <rule t="map">
+        <name>eth0</name>
+        <rule>ATTR{address}</rule>
+        <value>00:16:3e:ea:05:7c</value>
+      </rule>
+    </net-udev>
+    <routing t="map">
+      <ipv4_forward t="boolean">false</ipv4_forward>
+      <ipv6_forward t="boolean">false</ipv6_forward>
+    </routing>
+  </networking>
+  <ntp-client t="map">
+    <ntp_policy>auto</ntp_policy>
+    <ntp_servers t="list">
+      <ntp_server t="map">
+        <address>2.suse.pool.ntp.org</address>
+        <iburst t="boolean">true</iburst>
+        <offline t="boolean">false</offline>
+      </ntp_server>
+    </ntp_servers>
+    <ntp_sync>manual</ntp_sync>
+  </ntp-client>
+  <partitioning t="list">
+    <drive t="map">
+      <device>/dev/xvdb</device>
```

Comment:
Is it the right disk name here?

Reply:
Thanks. It's cloned from interactive installation, for VR this works well.

Reply:
From serial log, it is vda 
```
[  292.152996][ T2644] BTRFS info (device vda2): balance: start -dusage=0
[  292.153778][ T2644] BTRFS info (device vda2): balance: ended with status: 0
[  292.157013][ T2645] BTRFS info (device vda2): balance: start -dusage=5
[  292.157767][ T2645] BTRFS info (device vda2): balance: ended with status: 0
[  292.160817][ T2646] BTRFS info (device vda2): balance: start -dusage=10
[  292.161593][ T2646] BTRFS info (device vda2): balance: ended with status: 0
[  292.164825][ T2647] BTRFS info (device vda2): balance: start -musage=0 -susage=0
[  292.165677][ T2647] BTRFS info (device vda2): balance: ended with status: 0
[  292.168696][ T2648] BTRFS info (device vda2): balance: start -musage=3 -susage=3
[  292.169618][ T2648] BTRFS info (device vda2): relocating block group 22020096 flags system|dup
[  292.173288][ T2648] BTRFS info (device vda2): balance: ended with status: 0
[  293.313489][ T2740] BTRFS info (device vda2): scrub: started on devid 1
[  294.805122][ T2786] BTRFS warning (device vda2): failed to trim 5 block group(s), last error -512
[  294.812640][ T2786] BTRFS warning (device vda2): failed to trim 1 device(s), last error -512
[  300.177948][  T169] BTRFS info (device vda2): qgroup scan completed (inconsistency flag cleared)
```

Reply:
It is not supported to set 'vda', https://openqa.suse.de/tests/14981365#step/installation/44